AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/flowMisc.js

```diff
@@ -678,4 +678,54 @@ export function addSubnetSelectionDropDownView(siteInfo, diagProvider, flowMgr,
             dropdownView.dropdowns.length = 0;
             dropdownView.dropdowns.push(subscriptionDropdown);
         });
+}
+
+export async function checkDaaSExtApiAsync(diagProvider) {
+    var params = [];
+    params.push("api-version=2015-08-01");
+    var daasVersionInfo=await diagProvider.getDaaSExtApiAsync("daasversion",params);
+    var isDaasExtAccessible;
+    if(daasVersionInfo.status == "200") {
+        isDaasExtAccessible = true;
+    }
+    else {
+        isDaasExtAccessible = false;
+    }
+
+    var isDaasNew = false;
+    if(isDaasExtAccessible == true) {
+        var currentDaasVersion = daasVersionInfo.body.Version;
+        if (checkDassNewVersion(currentDaasVersion) >= 0) {
+            isDaasNew = true;
+        }
+        else {
+            isDaasNew = false;
+        }
+    }
+
+    return { "IsDaasExtAccessible": isDaasExtAccessible, "IsDaasNew": isDaasNew }
+}
+
+function checkDassNewVersion(currentDaasVersion){ 
```

Comment:
typo in function name.  Should be "DaaS" (preferable) or "Daas" and not "Dass"

Reply:
fixed